cmd/litcli/autopilot.go

```diff
@@ -39,10 +40,48 @@ var addAutopilotSessionCmd = cli.Command{
 	Name:      "add",
 	ShortName: "a",
 	Usage:     "Initialize an Autopilot session.",
-	Description: "Initialize an Autopilot session.\n\n" +
-		"   If set for any feature, configuration flags need to be " +
-		"repeated for each feature that is registered, corresponding " +
-		"to the order of features.",
+	Description: `
+	Initialize an Autopilot session.
+
+	If one of the 'feature-' flags is set for any 'feature', then that flag
+	must be provided for each 'feature'.
+
+	The rules and configuration options available for each feature can be
+	seen in the 'autopilot features' output.
+	
+	An example call reads:
+	
+	./litcli autopilot add --label=customRules \
+	--feature=SampleFeature \
```

Comment:
nit:
```suggestion
	--feature=AutoFees \
```

Probably makes more sense to mention Autofees instead.

Reply:
right, I updated to have an AutoFees example